src/core/dbus.c

```diff
@@ -737,6 +737,10 @@ static int bus_on_connection(sd_event_source *s, int fd, uint32_t revents, void
                 return 0;
         }
 
+        r = bus_register_malloc_status(bus);
+        if (r < 0)
+                log_info_errno(r, "Failed to register MallocStatus1, ignoring: %m");
```

Comment:
info? that doesn't sound right.

we usually use LOG_WARNING level for errors we are happy to ignore.